data-serving/data-service/schemas/cases.schema.json

```diff
@@ -213,110 +213,125 @@
+                        },
+                        "longitude": {
+                          "bsonType": "number",
+                          "minimum": -180.0,
+                          "maximum": 180.0
+                        }
+                      }
+                    }
+                  }
                 },
-                "geometry": {
+                "dateRange": {
                   "bsonType": "object",
                   "additionalProperties": false,
-                  "required": [
-                    "latitude",
-                    "longitude"
-                  ],
                   "properties": {
                     "_id": {
                       "bsonType": "objectId"
                     },
-                    "latitude": {
-                      "bsonType": "number",
-                      "minimum": -90.0,
-                      "maximum": 90.0
+                    "start": {
+                      "bsonType": "date"
                     },
-                    "longitude": {
-                      "bsonType": "number",
-                      "minimum": -180.0,
-                      "maximum": 180.0
+                    "end": {
+                      "bsonType": "date"
                     }
                   }
-                }
-              }
-            },
-            "dateRange": {
-              "bsonType": "object",
-              "additionalProperties": false,
-              "properties": {
-                "_id": {
-                  "bsonType": "objectId"
                 },
-                "start": {
-                  "bsonType": "date"
+                "purpose": {
+                  "enum": [
+                    "Business",
+                    "Leisure",
+                    "Family",
+                    "Other",
+                    "Unknown"
+                  ]
                 },
-                "end": {
-                  "bsonType": "date"
+                "method": {
+                  "enum": [
+                    "Bus",
+                    "Car",
+                    "Coach",
+                    "Ferry",
+                    "Plane",
+                    "Train",
+                    "Other",
+                    "Unknown"
+                  ]
                 }
               }
-            },
-            "purpose": {
-              "enum": [
-                "Business",
-                "Leisure",
-                "Family",
-                "Other",
-                "Unknown"
-              ]
-            },
-            "method": {
-              "enum": [
-                "Bus",
-                "Car",
-                "Coach",
-                "Ferry",
-                "Plane",
-                "Train",
-                "Other",
-                "Unknown"
-              ]
             }
+          },
+          "traveledPrior30Days": {
+            "bsonType": "bool"
+          },
+          "numLocations": {
+            "bsonType": "int"
           }
```

Comment:
Are these two fields derivable from the travel field? We should be cautious, IMO, about adding "convenience" fields to the core dataset.

Reply:
I agree. I have an open comment on the spec about this -- I think the idea might be that some datasets only have values for the "convenience" fields, and we don't want to lose that data -- but I'm not sure. I will definitely come back and clean it up if we change the spec.